arbitrary-users-patch/base_images

```diff
@@ -1,4 +1,4 @@
-che-python-3.6          centos/python-36-centos7:1
+che-python-3.7          python:3.7.4-slim
```

Comment:
+1 to changing the name to reflect Python 3.7, but we really need to consider what we do with `che-python-3.6`.

We currently do not have any deprecation procedure for old patched images; we can stop building `che-python-3.6` but any workspaces created from the old python stacks will continue to depend on this image. I'll open an issue to look into this.

Reply:
Thinking about it more, we should keep the `che-python-3.6` entry for now and remove it in a separate PR to allow us to ensure it doesn't cause problems.